src/themes/admin_future/modules/myapi/roles-add.tpl

```diff
@@ -0,0 +1,123 @@
+<form method="post" class="row g-3 ajax-submit" action="{$FORM_ACTION}" autocomplete="off" id="role" novalidate>
+    <div class="col-12">
+        <div class="card border-primary border-3 border-bottom-0 border-start-0 border-end-0">
+            <div class="card-header py-2 fw-medium fs-5">
+                {$LANG->getModule('api_role_properties')}
+            </div>
+            <div class="card-body">
+                <div class="row mb-3">
+                    <label for="role_title" class="col-sm-3 col-form-label text-sm-end">{$LANG->getModule('api_roles_title')} <span class="text-danger">*</span>:</label>
+                    <div class="col-sm-8 col-lg-6 col-xxl-5">
+                        <input type="text" id="role_title" name="role_title" value="{$DATA.role_title}" class="form-control w350" maxlength="250" required />
+                    </div>
+                </div>
+                <div class="row mb-3">
+                    <label for="role_description" class="col-sm-3 col-form-label text-sm-end">{$LANG->getModule('api_roles_description')}</label>
+                    <div class="col-sm-8 col-lg-6 col-xxl-5">
+                        <textarea class="form-control w350" id="role_description" name="role_description" rows="2" maxlength="250">{$DATA.role_description}</textarea>
+                    </div>
+                </div>
+                <div class="row mb-3">
+                    <label class="col-sm-3 col-form-label text-sm-end">{$LANG->getModule('api_role_type')}</label>
```

Comment:
@danlt150 thẻ label này và 3 cái bên dưới không có `for=`, Chrome sẽ thông báo `A <label> isn't associated with a form field.` nên để có sự chuyên nghiệp nhất cần:
- Thêm for=id cho nó
- Trong trường hợp nó không thể for=ID vì các input dạng checkbox, radio em hãy đổi thành div thay vì label